pages/pil/dashboard/index.js

```diff
@@ -1,15 +1,16 @@
 const page = require('../../../lib/page');
 const bodyParser = require('body-parser');
-const { has } = require('lodash');
+const { some, has } = require('lodash');
 
 module.exports = settings => {
   const app = page({
     ...settings,
     root: __dirname
   });
 
-  const profileOwnsModule = (profile, moduleId) => {
-    return !!profile.trainingModules.find(module => module.id === moduleId);
+  const profileOwnsModule = ({ trainingModules, exemptions }, moduleId) => {
+    let result = some(trainingModules.concat(exemptions), ['id', moduleId]);
+    return result;
```

Comment:
either `const` this, or just return result of the function

Reply:
~either const this, or~ just return result of the function